### DISCUSSION TITLE
How to skip specific CVEs in specific OS, such as "CVE-2023-27534" in Centos7.5

### DISCUSSION THREAD

**Discussion by m1ssya** [aquasecurity/trivy#6336] [Q&A]
### Question

How to skip specific CVEs in specific OS, such as "CVE-2023-27534" in Centos7.5
Trivy version 0.41.0.

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

None

### Operating System

Linux

**Comment by knqyf263** [comment]
It's not by OS, but you can filter by PURL. It will be included in v0.50.0, which will be released later today.
https://github.com/aquasecurity/trivy/pull/6178

PURL can include the disro.
https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#deb
